template_feed/Microsoft.DotNet.Common.ItemTemplates/content/BuildProps/.template.config/dotnetcli.host.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json.schemastore.org/dotnetcli.host",
+  "symbolInfo": {
+    "useArtifacts": {
+      "longName": "use-artifacts",
+      "shortName": ""
```

Comment:
the short name for this was `-ua` without this property. instead we disable it entirely.